zfit/models/kde.py

```diff
@@ -428,7 +428,7 @@ def _convert_init_data_weights_size(self, data, weights, padding):
         if callable(padding):
             data, weights = padding(data=data, weights=weights)
```

Comment:
```suggestion
            data, weights = padding(data=data, weights=weights, limits=limits)
```
Let's give here limits too  to be consistent